examples/asr/jasper_an4.py

```diff
@@ -23,61 +22,50 @@
 logging = nemo.logging
 
 
-def create_dags(jasper_params, args, nf):
-    vocab = jasper_params['labels']
-
-    # build train and eval model
-    train_dl_params = copy.deepcopy(jasper_params["AudioToTextDataLayer"])
-    train_dl_params.update(jasper_params["AudioToTextDataLayer"]["train"])
-    del train_dl_params["train"]
-    del train_dl_params["eval"]
+def create_dags(model_config_file, vocab, args, nf):
 
-    data_layer = nemo_asr.AudioToTextDataLayer(
-        manifest_filepath=args.train_dataset, labels=vocab, batch_size=args.batch_size, **train_dl_params,
+    # Create a data_layer for training.
+    data_layer = nemo_asr.AudioToTextDataLayer.import_from_config(
+        model_config_file, "AudioToTextDataLayer_train", overwrite_params={"manifest_filepath": args.train_dataset},
     )
 
     num_samples = len(data_layer)
-    steps_per_epoch = math.ceil(num_samples / (args.batch_size * args.iter_per_step * nf.world_size))
+    steps_per_epoch = math.ceil(num_samples / (data_layer.batch_size * args.iter_per_step * nf.world_size))
```

Comment:
This line is causing jasper_an4 CI test to fail as data_layer.batch_size returns 1, and steps_per_epoch, which is now incorrect, is used to calculate the learning rate policy

Reply:
yeah, self._batch_size wasn't set in the AudioDL at all, just pushed a fix, thanks @blisc !